index.html

```diff
@@ -17,7 +17,7 @@
         <canvas class="backgroundeffect" id="backgroundeffect"></canvas>
     </div>
     <div class="bgimg" id="bgimg"></div>
-    <div class="w3-sidebar w3-animate-left" style="width:270px;background-color:#0006;display:none" id="thesidebar">
+    <div class="w3-sidebar w3-animate-left" style="width:400px;background-color:#0006;display:none" id="thesidebar">
```

Comment:
For synchronized style, let's keep the `width:270px`.
As long as the style is updated in the `.core` repository, we should be good with 270.

Reply:
Generally no problem with suggested changes, one critique is sometimes the sidebar is too small. Whenever u go into sub settings in general, it will get cut off from the sidebar view.

Reply:
Make sense, let's go with your suggested 330.